targetplatform/SmartHome-Runtime.launch

```diff
@@ -24,7 +24,7 @@
efault,org.apache.commons.exec@default:default,org.apache.commons.httpclient@default:default,org.apache.commons.io@default:default,org.apache.commons.lang@default:default,org.apache.commons.logging@default:default,org.apache.commons.net@default:default,org.apache.felix.gogo.command@default:true,org.apache.felix.gogo.runtime@default:true,org.apache.felix.gogo.shell@default:true,org.apache.httpcomponents.httpclient*4.2.6.v201311072007@default:default,org.apache.httpcomponents.httpclient*4.3.6.v201411290715@default:default,org.apache.httpcomponents.httpcore*4.2.5.v201311072007@default:default,org.apache.httpcomponents.httpcore*4.3.3.v201411290715@default:default,org.eclipse.core.commands@default:default,org.eclipse.core.contenttype@default:default,org.eclipse.core.expressions@default:default,org.eclipse.core.filesystem@default:default,org.eclipse.core.jobs@default:default,org.eclipse.core.resources@default:default,org.eclipse.core.runtime@default:true,org.eclipse.emf.common@default:default,org.eclipse.emf.ecore.xmi@default:default,org.eclipse.emf.ecore@default:default,org.eclipse.equinox.app@default:default,org.eclipse.equinox.cm@3:true,org.eclipse.equinox.common@2:true,org.eclipse.equinox.console@default:true,org.eclipse.equinox.ds@2:true,org.eclipse.equinox.event@2:true,org.eclipse.equinox.http.servlet@default:default,org.eclipse.equinox.preferences@default:default,org.eclipse.equinox.registry@default:default,org.eclipse.equinox.security@default:default,org.eclipse.equinox.util@default:default,org.eclipse.jdt.core@default:default,org.eclipse.jetty.client@default:default,org.eclipse.jetty.continuation*9.2.9.v20150224@default:default,org.eclipse.jetty.deploy@default:default,org.eclipse.jetty.http*9.2.9.v20150224@default:true,org.eclipse.jetty.io*9.2.9.v20150224@default:default,org.eclipse.jetty.osgi.boot@default:true,org.eclipse.jetty.osgi.httpservice@default:default,org.eclipse.jetty.proxy@default:default,org.eclipse.jetty.security*9.2.9.v20150224@default:default,org.eclipse.jetty.server*9.2.9.v20150224@default:true,org.eclipse.jetty.servlet*9.2.9.v20150224@default:true,org.eclipse.jetty.util*9.2.9.v20150224@default:default,org.eclipse.jetty.webapp@default:default,org.eclipse.jetty.xml@default:default,org.eclipse.osgi.services@default:default,org.eclipse.osgi@-1:true,org.eclipse.text@default:default,org.eclipse.xtend.lib.macro@default:default,org.eclipse.xtend.lib@default:default,org.eclipse.xtext.common.types@default:true,org.eclipse.xtext.ecore@default:default,org.eclipse.xtext.smap@default:default,org.eclipse.xtext.util@default:default,org.eclipse.xtext.xbase.lib@default:default,org.eclipse.xtext.xbase@default:default,org.eclipse.xtext@default:default,org.json@default:default,org.jupnp@default:false,org.mapdb.mapdb@default:default,org.objectweb.asm@default:default,org.sat4j.core@default:default,org.sat4j.pb@default:default,org.slf4j.api@default:default,org.slf4j.jcl@default:default,org.slf4j.jul@default:default,org.slf4j.log4j@default:default"/>
```

Comment:
What has changed in the launch config? I would assume that this is unrelated to this PR?